### ISSUE TITLE
tideways.ini broken in php-tideways_5.0.1-1+ubuntu18.04.1+deb.sury.org+11_amd64.deb

### ISSUE THREAD

**Issue by gschoenberger** [oerdnj/deb.sury.org#1268]
**Describe the bug**
The current module config for tideways in the package `php-tideways as 5.0.1-1+ubuntu18.04.1+deb.sury.org+11` is broken. The ini file uses the wrong extension name, `tideways.so` instead of `tideways_xhprof.so`.

**To Reproduce**
1. Extract `tideways_5.0.1-1+ubuntu18.04.1+deb.sury.org+11.debian.tar.xz`
2. Open `tideways.ini`
3. See `extension=tideways.so`
Compared to the ".so" file installed:
1. Extract `php-tideways_5.0.1-1+ubuntu18.04.1+deb.sury.org+11_amd64.deb`
2. Open` /./usr/lib/php/20190902/`
3. See `tideways_xhprof.so`

The extension name in the .ini file does not match the ".so" file name.

**Expected behavior**
Ini file content:
```
extension=tideways_xhprof.so
tideways.auto_prepend_library=0
```
**Distribution (please complete the following information):**
 - OS: Ubuntu 18.04
 - Architecture: amd64
 - Repository: http://ppa.launchpad.net/ondrej/php/ubuntu/pool/main

**Package
ii  php-tideways                          5.0.1-1+ubuntu18.04.1+deb.sury.org+11                    amd64        Tideways PHP Profiler Extension

**Comment by gschoenberger** [comment]
Any chance to fix the INI file in the package?
The fix would be to change the line from
`extension=tideways.so`
to:
`extension=tideways_xhprof.so`

THX - Georg

**Comment by oerdnj** [comment]
Fixed in 5.0.1-2*
